Main/Source/devcons.cpp

```diff
@@ -300,7 +300,7 @@ void devcons::AddDevCmd(festring fsCmd, callcmd Call, festring fsHelp,bool bWiza
 {
   callcmd cc = Find(fsCmd.CStr());
   if(cc!=NULL) //TODO ignore if equal?
-    ABORT("command %s already set %x %x",fsCmd.CStr(),cc,Call);
+    ABORT("command %s already set %x %x",fsCmd.CStr(),(uint)cc,(uint)Call);
```

Comment:
Should use the `%p` specifier, no need to cast.

Reply:
Awesome, thanks @iology